README.md

```diff
@@ -43,7 +43,7 @@ libXinerama (for multi-monitor support)
 
 On debian based distros, all requeriments are installed with:
 
-> apt-get install xfce4-dev-tools build-essentialglib2libglib2.0-devxorg-devlibwnck-3-devlibclutter-1.0-devlibgarcon-1-0-devlibxfconf-0-devlibxfce4util-devlibxfce4ui-2-devlibxcomposite-devlibxdamage-dev libxinerama-dev
+> apt-get install xfce4-dev-tools build-essential glib2li bglib2.0-dev xorg-dev libwnck-3-dev libclutter-1.0-dev libgarcon-1-0-dev libxfconf-0-dev libxfce4util-dev libxfce4ui-2-dev libxcomposite-dev libxdamage-dev libxinerama-dev
```

Comment:
I think a space is at the wrong place. I believe you mean `glib2 libglib2.0-dev` instead of `glib2li bglib2.0-dev`.